City-of-Helsinki/helsinki-paatokset
# Pp 244 banner block

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-244

**How to test updates**

- Checkout branch
- Run `make fresh` (if you have an existing database dump)
- Run `make drush-updb` to make sure there aren't any pending database updates
- Make sure all modules and Drupal core is actually up-to-date: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
- Test that the site works and nothing weird was introduced in the platform config updates.

**How to test banner**

Create or go to some case and decision page. After all the content there should be a blue banner that has the text defined in the ticket.

Check the code changes, is it good that case and decision are preprocessed separately?

Changes from composer update

```
- Removing behat/mink-goutte-driver (v1.3.0)
  - Removing fabpot/goutte (v3.3.1)
  - Removing laminas/laminas-zendframework-bridge (1.2.0)
  - Upgrading composer/composer (2.1.12 => 2.2.3)
  - Locking composer/pcre (1.0.0)
  - Upgrading composer/semver (3.2.5 => 3.2.6)
  - Upgrading composer/xdebug-handler (2.0.2 => 2.0.3)
  - Upgrading consolidation/annotated-command (4.4.0 => 4.5.1)
  - Upgrading consolidation/log (2.0.2 => 2.0.4)
  - Upgrading consolidation/output-formatters (4.1.2 => 4.2.1)
  - Upgrading consolidation/robo (3.0.6 => 3.0.7)
  - Upgrading consolidation/self-update (2.0.0 => 2.0.3)
  - Upgrading consolidation/site-alias (3.1.1 => 3.1.3)
  - Upgrading consolidation/site-process (4.1.0 => 4.1.1)
  - Upgrading doctrine/annotations (1.13.1 => 1.13.2)
  - Upgrading drupal/core (9.2.9 => 9.3.0)
  - Upgrading drupal/core-composer-scaffold (9.2.9 => 9.3.0)
  - Upgrading drupal/core-dev (9.2.9 => 9.3.0)
  - Upgrading drupal/core-recommended (9.2.9 => 9.3.0)
  - Upgrading drupal/easy_breadcrumb (2.0.1 => 2.0.2)
  - Upgrading drupal/externalauth (1.4.0 => 2.0.0)
  - Upgrading drupal/hdbt_admin (1.3.17 => 1.4.3)
  - Upgrading drupal/helfi_platform_config (2.1.5 => 2.3.4)
  - Upgrading drupal/helfi_tpr (1.5.8 => 1.6.2)
  - Upgrading drupal/matomo (1.13.0 => 1.14.0)
  - Upgrading drupal/metatag (1.16.0 => 1.18.0)
  - Upgrading drupal/migrate_plus (5.1.0 => 5.2.0)
  - Upgrading drupal/openid_connect (2.0.0-alpha11 => 2.0.0-alpha12)
  - Locking drupal/publication_date (2.0.0-beta3)
  - Upgrading drupal/redirect (1.6.0 => 1.7.0)
  - Upgrading drupal/token (1.9.0 => 1.10.0)
  - Upgrading drush/drush (10.6.1 => 10.6.2)
  - Upgrading egulias/email-validator (2.1.25 => 3.1.2)
  - Upgrading elasticsearch/elasticsearch (v7.15.0 => v7.16.0)
  - Upgrading ezyang/htmlpurifier (v4.13.0 => v4.14.0)
  - Upgrading friends-of-behat/mink-browserkit-driver (v1.5.0 => v1.6.1)
  - Upgrading guzzlehttp/promises (1.4.1 => 1.5.1)
  - Upgrading guzzlehttp/psr7 (1.8.2 => 1.8.3)
  - Upgrading laminas/laminas-diactoros (2.6.0 => 2.8.0)
  - Upgrading laminas/laminas-escaper (2.7.0 => 2.9.0)
  - Upgrading laminas/laminas-feed (2.14.1 => 2.15.0)
  - Upgrading laminas/laminas-stdlib (3.3.1 => 3.6.1)
  - Upgrading masterminds/html5 (2.7.4 => 2.7.5)
  - Upgrading nikic/php-parser (v4.13.1 => v4.13.2)
  - Upgrading pear/pear-core-minimal (v1.10.10 => v1.10.11)
  - Upgrading phpspec/prophecy (1.14.0 => v1.15.0)
  - Upgrading phpunit/php-code-coverage (9.2.9 => 9.2.10)
  - Upgrading phpunit/php-file-iterator (3.0.5 => 3.0.6)
  - Upgrading psy/psysh (v0.10.9 => v0.10.12)
  - Upgrading ruflin/elastica (dev-master 40fac08 => dev-master 4124d20)
  - Upgrading seld/phar-utils (1.1.2 => 1.2.0)
  - Upgrading squizlabs/php_codesniffer (3.6.1 => 3.6.2)
  - Upgrading symfony-cmf/routing (2.3.3 => 2.3.4)
  - Upgrading symfony/console (v4.4.25 => v4.4.34)
  - Upgrading symfony/debug (v4.4.25 => v4.4.31)
  - Upgrading symfony/dependency-injection (v4.4.25 => v4.4.34)
  - Upgrading symfony/deprecation-contracts (v2.4.0 => v2.5.0)
  - Upgrading symfony/dom-crawler (v4.4.30 => v4.4.36)
  - Upgrading symfony/error-handler (v4.4.25 => v4.4.34)
  - Upgrading symfony/event-dispatcher (v4.4.25 => v4.4.34)
  - Upgrading symfony/event-dispatcher-contracts (v1.1.9 => v1.1.11)
  - Upgrading symfony/finder (v4.4.30 => v4.4.36)
  - Upgrading symfony/http-client-contracts (v2.4.0 => v2.5.0)
  - Upgrading symfony/http-foundation (v4.4.25 => v4.4.34)
  - Upgrading symfony/http-kernel (v4.4.25 => v4.4.35)
  - Upgrading symfony/lock (v4.4.33 => v4.4.36)
  - Upgrading symfony/mime (v5.3.0 => v5.4.0)
  - Upgrading symfony/phpunit-bridge (v5.3.11 => v5.4.0)
  - Upgrading symfony/polyfill-mbstring (v1.23.0 => v1.23.1)
  - Upgrading symfony/polyfill-php80 (v1.23.0 => v1.23.1)
  - Upgrading symfony/process (v4.4.25 => v4.4.35)
  - Upgrading symfony/psr-http-message-bridge (v2.1.0 => v2.1.2)
  - Upgrading symfony/routing (v4.4.25 => v4.4.34)
  - Upgrading symfony/serializer (v4.4.25 => v4.4.35)
  - Upgrading symfony/service-contracts (v2.4.0 => v2.5.0)
  - Upgrading symfony/translation (v4.4.25 => v4.4.34)
  - Upgrading symfony/translation-contracts (v2.4.0 => v2.5.0)
  - Upgrading symfony/validator (v4.4.25 => v4.4.35)
  - Upgrading symfony/var-dumper (v5.3.0 => v5.4.0)
  - Upgrading symfony/yaml (v4.4.25 => v4.4.34)
  - Upgrading twig/twig (v2.14.6 => v2.14.7)
  - Upgrading typo3/phar-stream-wrapper (v3.1.6 => v3.1.7)
```
